Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -559,6 +639,55 @@ - (NSTask *)childTask {
     return [self childTaskWithAppIds:_appId ? @[_appId] : @[]];
 }
 
+- (RLMApp *)createAppForPartition:(id<RLMBSON>)partition {
```

Comment:
This being `createAppForPartition:` confused me a bit, as I expected it to be doing something which actually required an app-per-partition, while it appears to only need an app per partition key type. I think making it `createAppForPartitionType:` instead would be a bit clearer.